src/main/resources/application.yml

```
@@ -26,7 +27,7 @@ fhir:
     psaObservationId: "https://fhir.diz.uk-erlangen.de/identifiers/obds/psa-observation-id"
     strahlentherapie-procedure-id: "https://bzkf.github.io/obds-to-fhir/identifiers/strahlentherapie-procedure-id"
     loinc: "http://loinc.org"
-    idco3Morphologie: "urn:oid:2.16.840.1.113883.6.43.1"
+    icdo3Morphologie: "http://terminology.hl7.org/CodeSystem/icd-o-3"
```

Das problem im snapshot test ist gerade noch, dass in den bisherigen DKTK Profilversionen das icdo3 system als OID fest-kodiert war. Die einfachste Lösung wäre wahrscheinlich erstmal ein neues property hier einfügen 

```suggestion
    icdo3Morphologie: "http://terminology.hl7.org/CodeSystem/icd-o-3"
    icdo3MorphologieOid: "urn:oid:2.16.840.1.113883.6.43.1"
```

und das dann in https://github.com/bzkf/obds-to-fhir/blob/63fe5cc233526f40e4d005fc1a59bc2687fc5c5f/src/main/java/org/miracum/streams/ume/obdstofhir/mapper/ObdsObservationMapper.java#L541 setzen. Dann sollten die snapshots unverändert bleiben.